packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -50,5 +50,10 @@ slot:not([name])::slotted(*) {
 }
 
 :host(:not([uswds])) #error-message {
-  margin-bottom: 0.8rem;
+  margin-bottom: 0;
 }
+
+:host(:not([uswds])) .date-container,
+:host .usa-memorable-date {
```

Comment:
You don't need to add :host here for `.usa-memorable-date`. We only need to leverage `:host` when we're trying to differentiate styles between the same selector that's being used in different ways (for example a class that's styled one way for v1 and styled another way for v3). In this case, `.usa-memorable-date` is only being used in v3 so it shouldn't need `:host`.

This should also be moved into the "v3 style area" as described in the follow comment.